spec/pane-view-spec.coffee

```diff
@@ -118,6 +118,14 @@ describe "PaneView", ->
         paneModel.activateItem(view2)
         expect(pane.itemViews.find('#view-2').length).toBe 1
 
+    describe "when the new activeItem implements ::getPath", ->
+      it "adds its file path to the active pane", ->
+        paneModel.activateItem(editor1)
+        expect(pane).toHaveAttr('data-active-path')
+      it "adds its file name to the active pane", ->
+        paneModel.activateItem(editor1)
+        expect(pane).toHaveAttr('data-active-name')
+
```

Comment:
It'd be good to have a test for the converse case as well: when the new active item does not implement ::getPath, it *removes* the attributes. You should be able to use a simple `div` element (`document.createElement("div")`) as the active-item for that case.

To make sure that the attributes are removed, you may want to test both adding and removing the attributes in a single `it`: first activate a `TextEditor`, then activate a `div`.